docs/website/developers/committers/release-process/publish.md

```diff
@@ -69,12 +74,15 @@ link in the top right (the credentials are the same as your Git and Jenkins cred
 Repositories" page, and tick the repository with the name starting `orgapachebrooklyn`. Click the **Release** button.
 Provide a description which includes the version, e.g. `Apache Brooklyn 0.7.0-incubating`.
 
+Note the staging repository may have a name like "080 RC2". You can double-check under the "content" that brooklyn-dist
```

Comment:
My point was just that the name will likely say "release candidate". When I did this step, I wasn't sure if I should see a staging repo for the 0.8.0-incubating that I had just pushed to `apache-dist-release-brooklyn`. I was not confident about pushing the RC staging repo until I'd checked with you @rdowner 

I'll reword it, but will not delete this paragraph.